docker-compose.yml

```
@@ -0,0 +1,53 @@
+services:
+  web:
+    build: .
+    command: sh -c 'poetry run python manage.py migrate && poetry run python manage.py runserver 0.0.0.0:8000'
+    volumes:
+      - .:/code
+    ports:
+      - '8000:8000'
+    depends_on:
+      - db
+      - celery
+      - celery-beat
+    env_file:
+      - .env
+
+  db:
+    image: postgres:16.0
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    env_file:
+      - ./.env
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U $POSTGRES_USER']
```

Нужно также указывать название базы (иначе будет падать warning лог, если они отличаются)
```suggestion
      test: ['CMD-SHELL', 'pg_isready -U $POSTGRES_USER -d $POSTGRES_DB']
```